### PR TITLE
Merge all succeeding Dependabot PRs

### PR DESCRIPTION
**Pull request by Vinnl**
After #318, Dependabot opened a whole bunch of PRs updating outdated PRs. Because merging them in order takes a long time (after every merge, the other PRs have to be updated again and have their pipelines run), this PR bundles all of those PRs whose pipelines succeeded. If the pipeline of this one still succeeds, we should be relatively safe in merging them all at once.

### CODE REVIEW

**Review by pmcb55 — APPROVED** [review]
LGTM

**Comment by NSeydoux** [comment]
What I usually do in similar situation is to comment `@dependabot merge` on the PRs. Dependabot automatically rebases the PRs, so it's not required to manually update PRs after each merge, and in the case where a previous merge fails the CI on a PR, dependabot  won't merge it. Are there additional reasons to do this work manually, besides the fact that letting dependabot do its thing can last for quite a while ?

**Comment by Vinnl** [comment]
@NSeydoux Aha, I was wondering why you always left them as comments :) No, the reason is that I did not know that Dependabot would automatically rebase and then merge them in a row. That's good to know.

**Pull request merged**
